### ISSUE TITLE
[Avatar] Include subheading in tooltip?

### ISSUE THREAD

**Issue by jpzwarte** [sl-design-system/components#3578]
### Provide a general summary of the issue here

This is a question about https://storybook.sanomalearning.design/?path=/story/media-avatar--overflow

The story is *not* an actual real world example. I've seen real world examples where student names are too long for the available width. That's why we initially added the tooltip for that.

The question here is whether the tooltip should include the subheading?

Also, the avatar should be keyboard focusable when it has a tooltip:

We need to research what to do in each of these cases.

short name
short subtitle

short name
very long su....

very l......
short stubtitle

very lo....
very long su....

### 👤 Your name

Jeroen

### 🧢 Your Product/Team

SLDS
